README.md

```diff
@@ -1,7 +1,19 @@
 OpenSSH Module
 --------------
 
-This module installes Openssh. It also installes firewall rules and check_mk tasks
+This module installes Openssh. It also installes firewall rules and check_mk tasks.
+
+If you define `openssh{sign_host_key{[something]}` it will also generate signes host keys.
+
+Use signed host keys
+-------------
+- Generate a password protected SSH CA
+- Place the CA file in `data/certs/ssh_ca` (default value)
+- Add password to your metadata (see below)
+- Link the `items/sign_host_keys.py` to your `[repo]/items` folder.
+- Install `sshkey-tools>=0.9` via `pip`.
```

Comment:
also: add to repos requirements.txt (then bw will throw an error, if it is not installed)